<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Osmium</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="back_button">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>←</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refresh_button">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>↻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="urlbar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusbar">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scroll_area">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scroll_area_widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>485</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="page_layout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOsmium">
    <property name="title">
     <string>Osmium</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionDOMInspector"/>
    <addaction name="actionCookie_Inspector"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionProxy"/>
    <addaction name="actionSend_cookies"/>
   </widget>
   <addaction name="menuOsmium"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionDOMInspector">
   <property name="text">
    <string>DOM Inspector</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionCookie_Inspector">
   <property name="text">
    <string>Cookie Inspector</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSend_cookies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Send cookies</string>
   </property>
  </action>
  <action name="actionProxy">
   <property name="text">
    <string>Configure Proxy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
